scalalib/worker/src/ZincWorkerImpl.scala

```diff
@@ -22,7 +22,7 @@ case class MockedLookup(am: File => Optional[CompileAnalysis]) extends PerClassp
 }
 
 class ZincWorkerImpl(compilerBridge: Either[
```

Comment:
Here too, a documentation comment explaining the various things that `compilerBridge` can be would be useful.